crux/src/Crux/Goal.hs

```diff
@@ -351,7 +352,12 @@ proveGoalsOnline sym opts ctxt explainFailure (Just gs0) =
          return (\_ -> return (), return (), return ())
        else
          prepStatus "Checking: " (countGoals gs0)
-     res <- withSolverProcess sym $ \sp ->
+
+     -- make sure online features are enabled
+     enableOpt <- getOptionSetting enableOnlineBackend (getConfiguration sym)
+     _ <- setOpt enableOpt True
+
+     res <- withSolverProcess sym (fail "Online solving not enabled!") $ \sp ->
               inNewFrame sp (go (start,end) sp goalNum gs0 nameMap)
```

Comment:
This should never happen, but it could be useful to clarify that it's an internal error if it ever does.